### PR TITLE
Feature/cookies page/cplp 713

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
Add cookie page with locals file

### CODE REVIEW

File: cx-portal/src/assets/locales/de/main.json [hunk 1]
```diff
@@ -184,7 +184,17 @@
       "message": "Fügen Sie hier Inhalte hinzu"
     },
     "cookiepolicy": {
-      "message": "Fügen Sie hier Inhalte hinzu"
+      "description1": "A cookie is a small piece of data (text file) that a website - when visited by a user - asks your browser to store on your device in order to remember information about you, such as your language preference or login information.",
```
Comment (oyo) [inline cx-portal/src/assets/locales/de/main.json:187]: should these texts be translated to german? Maybe you want to put that german part into the translator first:
https://portal-dev.demo.catena-x.net/translator

File: cx-portal/src/assets/locales/en/main.json [hunk 2]
```diff
@@ -184,7 +184,17 @@
       "message": "add content here"
     },
     "cookiepolicy": {
-      "message": "add content here"
+      "description1": "A cookie is a small piece of data (text file) that a website - when visited by a user - asks your browser to store on your device in order to remember information about you, such as your language preference or login information.",
+      "description2": "More specially, we use cookies for the following purposes:",
+      "description2_first": "Assisting you in navigation;",
+      "description2_second": "Assisting in login;",
+      "description2_third": "Remember your prefernces and settings;",
```
Comment (oyo) [inline cx-portal/src/assets/locales/en/main.json:191]: typo: preferences

**Review by oyo — CHANGES_REQUESTED** [review]
some changes, see comments

File: cx-portal/src/assets/locales/de/main.json (hunk 1 above)
Comment (jjeroch) [reply cx-portal/src/assets/locales/de/main.json:187]: @oyo  quick info - Nidhi and I aligned that she doesn't need to translate. I will do the translation, she will take over the technical implementation.

**Comment by jjeroch** [comment]
@oyo for me the question is more general. I would like to see the footer pages in a standalone translation file. Any other need from your side?

**Comment by oyo** [comment]
> @oyo for me the question is more general. I would like to see the footer pages in a standalone translation file. Any other need from your side?

Currently we have two different i18n namespaces/files `main` and `help`
https://github.com/catenax-ng/product-portal-frontend/tree/main/cx-portal/src/assets/locales/en

Maybe we rename the `help.json` to `footer.json` and bundle all footer resources there. We could also have an own file for every page but that would result in a lot of small files.

File: cx-portal/src/assets/locales/en/main.json (hunk 2 above)
Comment (nidhigarg-bmw) [reply cx-portal/src/assets/locales/en/main.json:191]: @oyo So what typo I have to follow exactly ? 
description2-first or description2First

**Comment by jjeroch** [comment]
@oyo yeah changing "help" to "footer" sounds actually good

**Comment by oyo** [comment]
@jjeroch okay do you want to create a ticket for that or shall I do? maybe @nidhigarg-bmw could take that task?

File: cx-portal/src/assets/locales/en/main.json (hunk 2 above)
Comment (oyo) [reply cx-portal/src/assets/locales/en/main.json:191]: Line 191: missing `e`
` "description2_third": "Remember your preferences and settings;",`

**Review by oyo — APPROVED** [review]
would be nice if you could fix that typo

**Review by jjeroch — CHANGES_REQUESTED** [review]
Hi @nidhigarg-bmw  please add all the cookie and imprint translations into the "help" file and change the name from "help" to "Footer". Thanks"

**Review by jjeroch — APPROVED** [review]

**Pull request merged**